src/datasets/sdks/ecosystem.ts

```diff
@@ -31,6 +34,9 @@ const LogoMap: Record<string, EcosystemElement['logo']> = {
   'ruby-no-fill.svg': RubySvg,
   'angular-no-fill.svg': AngularSvg,
   'rust-no-fill.svg': RustSvg,
+  'nextjs-no-fill.svg': NextjsSvg,
+  'svelte-no-fill.svg': SveltekitSvg,
```

Comment:
I think `svelte-no-fill.svg` and `nextjs-no-fill.svg` are unused now, so they can be removed from here and from the `static/img` folder.

Reply:
yea going to keep the logos as they might get used for docs / integrations later.